Taskfile.yml

```
@@ -356,8 +356,9 @@ tasks:
         --set crdPattern=*"
       - task: controller:wait-for-operator-ready
     vars:
+      DIR: "{{default .KIND_WORKLOAD_IDENTITY_PATH .DIR}}"
```

I find it clearer to use `|` syntax with `default` as the argument order trips me up otherwise (having the fallback value first is weird, even though required by the way templates work).

```suggestion
      DIR: "{{.DIR | default .KIND_WORKLOAD_IDENTITY_PATH}}"
```